src/app/content/highlights/reducer.ts

```diff
@@ -194,7 +194,14 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
       };
     }
     case getType(actions.focusHighlight): {
-      return {...state, currentPage: { ...state.currentPage, hasUnsavedHighlight: false, focused: action.payload }};
```

Comment:
@TomWoodward removing `hasUnsavedHighlight: false` from here fixes the bug Malar found when testing the discard changes modal. That property was added when the discard modal [was first implemented](https://github.com/openstax/rex-web/pull/485/files). Do you know why this would have been desirable? No tests are failing but I feel like this must break something.